gapic/schema/metadata.py

```diff
@@ -103,8 +102,13 @@ def __str__(self) -> str:
         return '.'.join(self.parent + (self.name,))
 
     @property
-    def is_external_type(self):
-        return not self.proto_package.startswith(self.api_naming.proto_package)
+    def is_proto_plus_type(self):
+        if self.proto_package.startswith(self.api_naming.proto_package) or (
```

Comment:
Simplify by returning the condition directly.

return if ....

Reply:
Fixed in 
https://github.com/googleapis/gapic-generator-python/pull/1596/commits/99a1ac8da58713d758b508f5980f7a940d228773